src/include/migraphx/quantization.hpp

```diff
@@ -41,11 +41,19 @@ struct program;
 MIGRAPHX_EXPORT void quantize_fp16(program& prog,
                                    const std::vector<std::string>& ins_names = {"all"});
 
+MIGRAPHX_EXPORT void quantize_8bits(program& prog,
```

Comment:
I dont think this needs to be declared in the header, its only used internally for `quantize_int8` and `quantize_fp8`.

Reply:
Done.